src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -73,6 +73,20 @@
                     <attribute name="groups">cart:add_item</attribute>
                 </attribute>
             </itemOperation>
+
+            <itemOperation name="remove_item">
+                <attribute name="security">is_granted('IS_AUTHENTICATED_ANONYMOUSLY')</attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">Remove Item</attribute>
+                </attribute>
+                <attribute name="method">PATCH</attribute>
```

Comment:
```suggestion
                <attribute name="method">DELETE</attribute>
```

Reply:
Delete can't has a body. In our case we need add product code to delete, I thing that in this case we cant add DELETE, https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request

<img width="1347" alt="Screenshot 2020-06-30 at 11 22 53" src="https://user-images.githubusercontent.com/39232096/86109128-5293dd80-bac4-11ea-8135-6587bd2ec645.png">

Reply:
Oh, but this IRI structure is wrong. We should rather have `/new-api/orders/{id}/items/{itemId}` with DELETE method on it.